README.md

```diff
@@ -1,2 +1,148 @@
+It also includes some config linters for (`HTML` / `CSS3` / `JavaScript`) in the `.github` folder. 
+
+📌 **Live Demo:**
+- See the project live from [here](https://obibaadoma.github.io/To-Do-List/dist/)
+
+📌 **Tech Stack:**
+- Page strucutre is built with `HTML5`
+- Styling and alignment built with `CSS3`
+- Interaction and dynamic content built with `JavaScript`
+
+📌 **Key Features:**
+- Single page app where all content is loaded dynamically
+- Three navigation links (list / add / contact)
+- Transition effect in content loading
+- Active content link is highlighted
+- Displays a list of books with title and author values
+- Inserts new book with a form input of title and author fields
+- Removes any book item with a relevant remove button
+- Uses an array of objects for handling books data
+- Stores the books array when adding or removing locally
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- INSTRUCTIONS -->
+
+<a name="instructions"></a>
+
+# 🛠️ Instructions
+
+You can easily download or fork this repository and work on it immadiately!
+
+📌 **Prerequisites:**
+- No prior requirements needed!
+
+📌 **Setup:**
+To get a local copy up and running follow these simple example steps.
+
+Open terminal
+Change the directory you want put the repo
+Then run:
+ i. https://obibaadoma.github.io/To-Do-List/dist/ 
+ii. cd repo-name/
+
+📌 **Deployment:**
+- You can deploy this project by uploading three files (`index.html` / `style.css` / `script.js`) to a live server.
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- AUTHORS -->
+
+<a name="authors"></a>
+
+# 👥 Authors
+
+📌 **Derek:**
+- [GitHub](https://github.com/obibaadoma)
+- [Twitter](https://twitter.com/mobibaadoma)
+- [LinkedIn](https://www.linkedin.com/in/derek-akrasi-konadu-187453151/)
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- FUTURE -->
+
+<a name="future"></a>
+
+# 🔭 Future
+
+Some additional features I may implement in the project:
+- [ ] Backend and database integration
+- [ ] Admin panel for content management
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- CONTRIBUTIONS -->
+
+<a name="contributions"></a>
+
+# 🤝🏻 Contributions
+
+Wish to contribute to this project?
+<br/>
+Contributions, issues, and feature requests are more than welcome!
+<br/>
+Feel free to check the [issues](../../issues) page too.
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- SUPPORT -->
+
+<a name="support"></a>
+
+# ⭐️ Support
+
+Like this project? Show your support by starring!
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+<a name="acknowledgements"></a>
+
+# 🙏🏻 Acknowledgements
+
+I thank everyone at **Microverse** for guiding me through this project.
+
+<p align="right"><a href="#title">back to top</a></p>
+
+<!-- LICENSE -->
+
+<a name="license"></a>
+
+# 📝 License
+
+This project is [MIT](LICENSE.md) licensed.
```

Comment:
- [ ] Great job adding a license file. Please kindly ensure you link it well as your MIT links is still indicating as broken. 

![Screenshot (1241)](https://github.com/obibaadoma/To-Do-List/assets/65298048/629cf494-96aa-4b12-9ebd-f700b553e71b)